tests/models/test_cli.py

```diff
@@ -22,7 +22,7 @@
 from mlflow.utils.file_utils import TempDir, path_to_local_file_uri
 from mlflow.utils.environment import _mlflow_conda_env
 from mlflow.utils import PYTHON_VERSION
-from tests.models import test_pyfunc
+from tests.models import dummy_pyfunc
```

Comment:
`tests.models.test_pyfunc.py` doen't contain any test function. Running `pytest` against this file exits with 5 (meaning: `No tests were collected`, [link](https://docs.pytest.org/en/stable/usage.html#possible-exit-codes))